### DISCUSSION TITLE
Headers are not mapped to "Page Contents" side section, when data incoming from external API

### DISCUSSION THREAD

**Discussion by robertogbezerra** [requarks/wiki#6836] [Error / Bug Report]
The is showing the data from an external API, but is not rendering it into "Page Contents" side section. I´m using using Javascript to inject this data following the Wiki.js documentation.

![imagem](https://github.com/requarks/wiki/assets/35476079/9a0920e7-01f5-4b41-9131-73c13d58816b)

**Comment by NGPixel** [comment]
The table of contents is generated at render time, not on page load. If you inject content client-side, that won't work.

**Comment by robertogbezerra** [comment]
Is there any way to integrate with an external api or internal database to
get data dynamically?



Em ter., 21 de nov. de 2023 5:02 PM, Nicolas Giard ***@***.***>
escreveu:

**Reply by NGPixel** [reply]
Not at the moment

**Comment by robertogbezerra** [comment]
Ok, thanks for the reply.

Em qua., 22 de nov. de 2023 6:52 AM, Nicolas Giard ***@***.***>
escreveu:
